src/components/workspace/NewItem.svelte

```diff
@@ -79,6 +79,7 @@
         const body = Object.fromEntries(
           new URLSearchParams(new FormData(target)).entries()
         );
+        if (body.citation) body.citation = {default: body.citation}
```

Comment:
I don't know if that was the right place to do that.

Reply:
Long term, the update, create, and delete functions for a source should be consolidated into a Source model but we aren't there yet so, in the meantime, this place is as good as any 🙂